### PR TITLE
Vf legacy

### PR DESCRIPTION
**Pull request by AshaKharat**
Please review the scripts as part of Subscription Code Merge Review.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (nivethan-moolya) [inline .gitignore:1]: Do Not commit ANY DS_Store file

File: AppLocators/Android/A_FundTransferLocators.robot [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+${e_withdrawalProcess} =  Withdrawal Process
+${e_settledCash} =  Settled Cash
+${e_settledCashAmt} =  $0.00
+${e_amtToWithdraw} =  Amount to Withdraw
+${e_withdrawProcessNote} =  Note: Drivewealth charges a $11 withdrawal fee. Intermediary banks may charge additional fees on top of this amount.
+${e_proceedBtn} =  Proceed
+
+# LOCATORS
+${vf_A_transfer} =  xpath=//*[@text='Transfer']
+${vf_A_imgInTransfer} =  xpath=//*[@class='android.view.ViewGroup' and ./parent::*[@class='android.view.ViewGroup'] and (./preceding-sibling::* | ./following-sibling::*)[@text='Cash Balance']]
```
Comment (nivethan-moolya) [inline AppLocators/Android/A_FundTransferLocators.robot:86]: Is this a healthy element to use? especially in different environments?

File: Resources/Utils/Android/A_ForgotPassword.robot [hunk 3]
```diff
@@ -51,32 +52,30 @@ User Enters Invalid Email
...
+    Sleep  2s
     Enter New Password  ${e_newPwd}
+    Close Android Keyboard
+    Sleep  2s
     Enter New Password Again  ${e_newInvalidPwd}
+    Sleep  2s
     Close Android Keyboard
-    Log to Console  Closed Keyboard
-    Wait For Element Visibility On Android  ${vf_A_changePwd}
+    Sleep  5s
```
Comment (nivethan-moolya) [inline Resources/Utils/Android/A_ForgotPassword.robot:79]: Too many sleeps, please use Wait** keyword for any available Element or Text in the screen

File: Resources/Utils/Android/A_FundTransfer.robot [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+    Log to Console  Verified new entry created under Incomplete Deposits!
+
+Verify User Is Navigated To Upload The Wire Receipt Screen
+    Wait For Page Conatin Element  ${e_uploadWireReceipt}  5s
+    Verify Page Conatin Text  ${e_uploadWireReceipt}
+
+Upload Wire Receipt And Verify Success Message
+    Wait And Click Element On Android  ${vf_A_chooseFromLib}
+    Wait And Click Element On Android  ${vf_A_driversLicenseBackImg}
+    Wait For Page Conatin Element  ${e_receiptUploadSuccessMsg}  10s
```
Comment (nivethan-moolya) [inline Resources/Utils/Android/A_FundTransfer.robot:115]: Collect all WAITS** and write them in Common.robot file with fixed timeout as 20 or 30s
and call the same method. Requesting not to use any Waits directly in the script.

**Review by nivethan-moolya — COMMENTED** [review]
1. Too many sleeps
2. Make a common method for all waits* (for text, for an element so & so with 20 or 30s as common)
3. There are Conflicting files, please resolve conflicts as a priority.

File: AppLocators/Android/A_FundTransferLocators.robot (hunk 2 above)
Comment (AshaKharat) [reply AppLocators/Android/A_FundTransferLocators.robot:86]: The same locator worked on staging, legacy & refactored build. Hence, kept it same way.

File: Resources/Utils/Android/A_ForgotPassword.robot (hunk 3 above)
Comment (AshaKharat) [reply Resources/Utils/Android/A_ForgotPassword.robot:79]: Fixed

File: Resources/Utils/Android/A_FundTransfer.robot (hunk 4 above)
Comment (AshaKharat) [reply Resources/Utils/Android/A_FundTransfer.robot:115]: Fixed

**Comment by AshaKharat** [comment]
Fixed all the Code changes

**Pull request closed without merging**